packages/doc/src/stories/Modal.stories.js

```diff
@@ -43,15 +43,24 @@ const ModalWithOverlayTemplate = (args, { argTypes }) => ({
     showModal: false,
   }),
   methods: {
-    open () {
-      this.showModal = true
-    }
+    open() {
+      this.showModal = true;
+    },
   },
   template: `
     <div>
       <mkr-overlay :opened="showModal" />
       <mkr-modal v-bind="$props" v-model="showModal">
-        <span>Hello World</span>
+      <div style="display: flex; flex-direction: column; align-items: start;">
+          <h1>Hello world</h1>
+          <input type="text" />
+          <button>Button</button>
+          <textarea></textarea>
+          <select>
+            <option>1</option>
+            <option>2</option>
+          </select>
```

Comment:
Could use mkr components here... 😄

Reply:
You're right ! way nicer with mikado components